uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -116,18 +166,75 @@ public void discoverCoreDeviceBroker(String string) {
         //@TODO Implement method
     }
```

Comment:
How connection we checked will be identified ?

Reply:
Currently we agreed that connectionId == iot thing id